client/components/profilePage/recommendedBooks/recommendedBooksQuery.js

```diff
@@ -0,0 +1,199 @@
+  'childrens-middle-grade-paperback',
+  'paperback-books',
+  'picture-books',
+  'series-books',
+  'young-adult',
+  'young-adult-e-book',
+  'young-adult-hardcover',
+  'young-adult-paperback',
+  'animals',
+  'audio-fiction',
+  'audio-nonfiction',
+  'business-books',
+  'celebrities',
+  'crime-and-punishment',
+  'culture',
+  'education',
+  'espionage',
+  'expeditions-disasters-and-adventures',
+  'fashion-manners-and-customs',
+  'food-and-fitness',
+  'games-and-activities',
+  'graphic-books-and-manga',
+  'hardcover-business-books',
+  'health',
+  'humor',
+  'indigenous-americans',
+  'relationships',
+  'mass-market-monthly',
+  'middle-grade-paperback-monthly',
+  'paperback-business-books',
+  'family',
+  'hardcover-political-books',
+  'race-and-civil-rights',
+  'religion-spirituality-and-faith',
+  'science',
+  'sports',
+  'travel',
+  'young-adult-paperback-monthly'
+]
+
+const selectBestSellers = [];
+const trendingBooks = [];
+
+// Queries top books from NYT Books API lists; using NYT response, queries Google Books API for details
+// const getRecommendedBooks = async () => {
+//   const trendingBooks = [];
+//   const selectBestSellers = [];
+//   try {
+//     nytSelectedLists.forEach(async (currentList) => {
+//       await axios
+//         .get('http://localhost:3000/book/bestsellers', { params: { apikey: '0u7qPMbjUFiEb2b8zkQlBRo8TSsk2Avm', list: currentList }})
+//         .then(topTitle => {
+//           selectBestSellers.push(parseInt(topTitle.data.results.books[0].primary_isbn10));
+//           console.log(selectBestSellers);
+//         })
+//         .catch(error => { console.error(`Could not retrieve NYT ${currentList}: `, error.message) })
+//     });
+//     try {
+//       selectBestSellers.forEach( (isbn) => {
+//         axios.get('http://localhost:3000/book/trending',
+//             { params: { bookIsbn: isbn, apikey: 'AIzaSyArmjVhT3A34knGwXM3oQ-OhQzLrJTjIRA' }})
+//           .then(bookInfo => {
+//             console.log('bookInfo: ', bookInfo)
+//             trendingBooks.push(bookInfo);
+//             console.log('trending books ', trendingBooks)
+//           })
+//           .catch(error => { console.error(`Could not retrieve Google API data for ${isbn}: `, error.message); })
+//       })
+//     } catch (error) {
+//       console.log(`Google API request failed: `, error.message);
+//     }
+//     console.log('these are the trending books: ', trendingBooks);
+//   } catch(error) {
+//     console.error(`Trending books retrieval failed overall: `, error.message);
+//   }
+// }
+
+// getRecommendedBooks();
+
+const getRecommendedBooks = async () => {
+  // try {
+    selectBestSeller = []
+    return new Promise((resolve, reject) => {
+      return resolve(
+      nytSelectedLists.forEach(async (currentList) => {
+        await axios
+        .get('http://localhost:3000/book/bestsellers', { params: { apikey: '0u7qPMbjUFiEb2b8zkQlBRo8TSsk2Avm', list: currentList }})
```

Comment:
Don't think this api key should be here. Move to .env? Also you can just set the url to /book/bestsellers. From my understanding, this should default to the url the server is listening on